nikola/plugins/compile/rest/chart.py

```diff
@@ -73,52 +73,68 @@ class Chart(Directive):
     has_content = True
     required_arguments = 1
     option_spec = {
-        "copy": directives.unchanged,
+        "box_mode": directives.unchanged,
+        "classes": directives.unchanged,
         "css": directives.unchanged,
+        # "defs": directives.unchanged,
```

Comment:
If you can get it to work, you can just push one more commit to your fork with this uncommented.